google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -94,23 +103,34 @@ protected SchemaMismatchedException(Status grpcStatus, String name) {
     }
   }
 
-  /** Offset already exists. */
+  /**
+   * Offset already exists. This happens when appending with offset that is less than the current
+   * end of the offset on the Stream. ou can also choose to append to the end of the Stream by not
+   * specifying offset.
+   */
   public static final class OffsetAlreadyExists extends StorageException {
     protected OffsetAlreadyExists(
         Status grpcStatus, String name, Long expectedOffset, Long actualOffset) {
       super(grpcStatus, name, expectedOffset, actualOffset, ImmutableMap.of());
     }
   }
 
-  /** Offset out of range. */
+  /**
+   * Offset out of range. This happens when appending with offset that is larger than the current
+   * end of the offset on the Stream. You can also choose to append to the end of the Stream by not
+   * specifying offset.
+   */
   public static final class OffsetOutOfRange extends StorageException {
     protected OffsetOutOfRange(
         Status grpcStatus, String name, Long expectedOffset, Long actualOffset) {
       super(grpcStatus, name, expectedOffset, actualOffset, ImmutableMap.of());
     }
   }
 
-  /** Stream is not found. */
+  /**
+   * Stream created by CreateWriteStream is not found. You need to recreate a Stream using
```

Comment:
The stream is not found.  Possible causes include incorrectly specifying the stream identifier or attempting to use an old stream identifier that no longer exists.  You can invoke CreateWriteStream to create a new stream.